test/size.txt

```diff
@@ -1,38 +1,38 @@
- 71.21 kB   |    10.46 kB   |    4.03 kB   |   v0/amp-soundcloud-0.1.js
- 79.18 kB   |    12.95 kB   |    4.68 kB   |   v0/amp-springboard-player-0.1.js
-132.13 kB   |    24.59 kB   |    8.84 kB   |   v0/amp-twitter-0.1.js
-278.81 kB   |    63.63 kB   |   18.79 kB   |   v0/amp-user-notification-0.1.js
- 71.14 kB   |    10.37 kB   |       4 kB   |   v0/amp-vimeo-0.1.js
- 70.67 kB   |    10.23 kB   |    3.95 kB   |   v0/amp-vine-0.1.js
- 81.63 kB   |    13.26 kB   |    4.91 kB   |   v0/amp-youtube-0.1.js
-135.66 kB   |    36.89 kB   |   12.36 kB   |   current-min/f.js / current/integration.js
\ No newline at end of file
+611.74 kB   |   172.68 kB   |   47.77 kB   |   v0.js / amp.js
+352.88 kB   |    86.27 kB   |   25.62 kB   |   v0/amp-access-0.1.js
+ 27.07 kB   |     9.35 kB   |    3.63 kB   |   v0/amp-accordion-0.1.js
+372.16 kB   |   100.11 kB   |   31.51 kB   |   v0/amp-analytics-0.1.js
+ 87.25 kB   |    14.26 kB   |    5.32 kB   |   v0/amp-anim-0.1.js
+ 84.09 kB   |    13.66 kB   |    5.18 kB   |   v0/amp-audio-0.1.js
+ 78.94 kB   |    12.77 kB   |    4.69 kB   |   v0/amp-brid-player-0.1.js
+ 94.63 kB   |    16.66 kB   |    6.08 kB   |   v0/amp-brightcove-0.1.js
+191.06 kB   |    38.85 kB   |   11.36 kB   |   v0/amp-carousel-0.1.js
+ 71.52 kB   |    10.68 kB   |    4.12 kB   |   v0/amp-dailymotion-0.1.js
+ 81.85 kB   |    10.84 kB   |    4.16 kB   |   v0/amp-dynamic-css-classes-0.1.js
+131.66 kB   |    24.25 kB   |     8.7 kB   |   v0/amp-facebook-0.1.js
+ 34.58 kB   |    10.76 kB   |    4.07 kB   |   v0/amp-fit-text-0.1.js
+ 88.51 kB   |    14.11 kB   |    4.93 kB   |   v0/amp-font-0.1.js
+123.55 kB   |    23.33 kB   |    8.21 kB   |   v0/amp-iframe-0.1.js
+216.98 kB   |    48.58 kB   |    13.8 kB   |   v0/amp-image-lightbox-0.1.js
+ 87.44 kB   |    14.32 kB   |    5.23 kB   |   v0/amp-instagram-0.1.js
+ 81.11 kB   |    12.78 kB   |    4.78 kB   |   v0/amp-install-serviceworker-0.1.js
+ 78.37 kB   |     12.2 kB   |    4.57 kB   |   v0/amp-jwplayer-0.1.js
+ 92.36 kB   |    16.26 kB   |    6.03 kB   |   v0/amp-kaltura-player-0.1.js
+136.78 kB   |    21.81 kB   |    6.91 kB   |   v0/amp-lightbox-0.1.js
+276.88 kB   |    61.52 kB   |   18.13 kB   |   v0/amp-list-0.1.js
+143.99 kB   |    44.74 kB   |   15.59 kB   |   v0/amp-mustache-0.1.js
+120.43 kB   |    30.61 kB   |    9.39 kB   |   v0/amp-pinterest-0.1.js
+ 70.64 kB   |    10.24 kB   |    3.95 kB   |   v0/amp-reach-player-0.1.js
+  95.3 kB   |    17.16 kB   |     6.1 kB   |   v0/amp-sidebar-0.1.js
+105.86 kB   |    17.94 kB   |    6.33 kB   |   v0/amp-slides-0.1.js
+114.12 kB   |    25.01 kB   |    8.89 kB   |   v0/amp-social-share-0.1.js
+ 71.21 kB   |    10.43 kB   |    4.02 kB   |   v0/amp-soundcloud-0.1.js
+ 79.18 kB   |    12.93 kB   |    4.67 kB   |   v0/amp-springboard-player-0.1.js
+132.13 kB   |    24.38 kB   |    8.74 kB   |   v0/amp-twitter-0.1.js
+278.81 kB   |    62.34 kB   |   18.34 kB   |   v0/amp-user-notification-0.1.js
```

Comment:
thats a good question. it looks like it just has a lot of small dependencies on core that are not abstracted out into services.

notable ones:
- url-replacement (surprised how big this is)
- style
- xhr
- storage
- log (this is common and the easiest one to fix across all extensions)